lib/sdk.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2016 Kinvey Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+'use strict';
+
+const dataLink = require('./service/dataLink');
+const businessLogic = require('./service/businessLogic');
+const moduleGenerator = require('./service/moduleGenerator');
+
+const receiver = require('kinvey-code-task-runner');
+
+const generateService = (callback) => {
```

Comment:
This shouldn't be an arrow function or a variable.  The preferred way is for functions that are named to use function declarations 

e.g.
```
function generateService(callback) {
}
```

This provides for better stack traces and is more performant.  Arrow functions are good for inline, anonymous functions.  The Coffeescript way of `<variable> =  (stuff) ->` isn't the correct pattern for es6.